tests/functional/firefox/test_notes.py

```diff
@@ -0,0 +1,113 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.accounts import FirefoxNotesPage
+
+
+@pytest.mark.nondestructive
+def test_firefox_desktop_release_page(base_url, selenium):
+    #firefox/65.0/releasenotes/
```

Comment:
If you want to pass this into a test to open a specific URL / version, you could try using a parameterized test to pass in a slug to the URL. You can see an existing example of that [here](https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/features/test_features.py#L10-L18) and [here](https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/features/feature.py#L13).